src/__tests__/mcp-server.test.ts

```
@@ -0,0 +1,524 @@
+          },
+        },
+        async ({ errorType }) => {
+          switch (errorType) {
+            case 'validation':
+              throw new Error('Validation failed: Invalid input format');
+            case 'network':
+              throw new Error('Network error: Failed to connect to FreeScout API');
+            case 'notfound':
+              throw new Error('Not found: Ticket #123 does not exist');
+            default:
+              const output = { result: 'success' };
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)